voorhoede/tudelft-spacefinder
# Remove the row for "Type locatie" from the filters

# Changes

- Removes the location type from the filters.
- Removes `studyType` from the project, except for the space detail card.

# Associated issue

Partially resolves https://trello.com/c/RYTUsSZt/52-determine-which-icons-and-filters-are-still-relevant

# How to test

1. Open preview link.
2. Open the filter menu and note that the filter row for type of location is gone ("instruction room" or "onderwijszaal" are not visible any more).
3. Make sure that the icon is still visible in the space card.

# Checklist
